veloroutes_voies_vertes/processing/chargement/load_layers.py

```diff
@@ -168,12 +190,24 @@ def processAlgorithm(self, parameters, context, feedback):
                 output_layers.append(result.id())
 
         # add attribute tables
+        style = self.parameterAsBool(parameters, self.STYLE, context)
+        save_style = self.parameterAsBool(parameters, self.SAVE_STYLE, context)
         for x in tables_name:
             if not context.project().mapLayersByName(x):
                 result = self.initLayer(context, uri, schema, x, None, "")
                 if not result:
                     feedback.pushInfo("La couche " + x + " ne peut pas être chargée")
                 else:
+                    msg_style = ""
                     output_layers.append(result.id())
+                    if style and save_style:
+                        msg_style = "Style for " + x + " successfully loaded and saved in database"
+                        result.saveStyleToDatabase(result.name(), 'default description', True, '')
+                    elif style and not save_style:
+                        result.loadNamedStyle(resources_path("qml", x + ".qml"))
+                        msg_style = "Style for " + x + " successfully loaded"
+                    elif not style and save_style:
+                        msg_style = "Style for can't sved in database if not loaded"
```

Comment:
Style for **segment** can't **be** s**a**ved. 

3 fautes dans la ligne

Mais l'extension étant franco-française, je pense que l'on peut rester en français concernant l'UX (User eXperience)

Reply:
Comme tous les messages sont en anglais j'ai continué